src/vcpkg/platform-expression.cpp

```diff
@@ -223,43 +308,97 @@ namespace vcpkg::PlatformExpression
                     // platform-expression-simple
                     return std::make_unique<ExprImpl>(ExprKind::op_not, expr_simple());
                 }
+                else if (cur() == 'n')
+                {
+                    std::string name = match_zero_or_more(is_identifier_char).to_string();
+
+                    // "not"
+                    if (name == "not")
+                    {
+                        // optional-whitespace,
+                        skip_whitespace();
+                        // platform-expression-simple
+                        return std::make_unique<ExprImpl>(ExprKind::op_not, expr_simple());
```

Comment:
I think this disagrees with the EBNF above -- it says the continuation should be `platform-expression-unary-keyword-operand`, not `platform-expression-simple`

Reply:
You are correct. I have fixed the comments.